### ISSUE TITLE
Notebooks

### ISSUE THREAD

**Issue by suzana-ilic** [Machine-Learning-Tokyo/d2l.ai#4]
hi guys! a quick suggestion, please make sure that the notebook titles are consistent and don't contain "copy", e.g. d2l_CNN, d2l_RNN, d2l_attention_1, d2l_attention_2, ... 

<img width="441" alt="Screenshot 2021-05-07 at 08 11 02" src="https://user-images.githubusercontent.com/27798583/117411710-2cc13b80-af14-11eb-9afa-fc621e000f71.png">

Ideally it should also not be run when uploaded (to avoid lots of log, see screenshot of !pip install stuff). 

<img width="1060" alt="Screenshot 2021-05-07 at 08 11 48" src="https://user-images.githubusercontent.com/27798583/117411857-59755300-af14-11eb-9fd7-be1a2b15abeb.png">

Also, I've seen that one live coding notebook (with blank code sections) was uploaded, we should upload the templates with the full code – would you agree?

Thanks you @pwuethri @devanshkv!

**Comment by devanshkv** [comment]
I agree! @KshitijAggarwal and I will take care of this for the Americas/EMEA sessions page.

**Comment by pwuethri** [comment]
@suzana-ilic Thank you very much for pointing out this issue. I will fix the link to point to the correct notebook

**Comment by suzana-ilic** [comment]
Thank you very much guys!
